definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-referral-nonprod.json

```diff
@@ -108,5 +108,137 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "GeneralReferralPayment",
+    "CaseFieldID": "GeneralReferralPaymentDetails",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageLabel": "General referral payment",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
```

Comment:
I might be wrong, but last time I was talking to Jeremy he decided "Check your answer" page is on by default.

Reply:
https://tools.hmcts.net/jira/browse/DIV-6166?focusedCommentId=867092&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-867092
![Screenshot 2020-10-20 at 13 43 05](https://user-images.githubusercontent.com/30896346/96587419-35d11f00-12da-11eb-95ce-b5290672547b.png)

Reply:
"1 week ago" vs 1 day ago :)

Reply:
Pardon?

Reply:
Jeremy's comment is from a week ago and I have talked to him on Monday :)

Reply:
If its not in the ticket, its not being asked of me. If Jeremy wants to add it to the ticket, he can. Otherwise, i have delieved what was expected from the ticket